### ISSUE TITLE
Create a policy of permanently deleting empty issues

### ISSUE THREAD

**Issue by josh-hemphill** [goldbergyoni/nodebestpractices#1093]
One of the only reasons I can think of for creating empty issues is to cheat Hacktober events, or the like. So by permanently deleting them, you ensure that those creating bots don't receive any benefit from opening empty issues here. To make sure we're not closing issues from people who are still just figuring it out, we could set some number of hours before it gets deleted.
It seems like something there should be a github bot for; I'll look into that.

**Comment by rluvaton** [comment]
Great idea, have it happened again?

**Comment by josh-hemphill** [comment]
Yeah. Just closed another one with an account that had only created empty issues on multiple repositories.

**Comment by rluvaton** [comment]
I'm not sure it's related to October events as it hasn't happened in October at all in my knowledge:

Empty Issues:
- #1092 
- #1068
- #981
- #980
- #764

**Comment by rluvaton** [comment]
Maybe there is some YouTube tutorial on how to submit issues (similar to the YouTube tutorial that "teaches" people how to contribute to open-source and added an example of adding "Amazing" to the project name in the readme)

**Comment by josh-hemphill** [comment]
Yeah. Looking at popular github bots, it seems like it would be pretty easy to set a bot that will comment with resources on how to write a good issue, along with a warning that if the issue isn't updated, it will be automatically closed/deleted after a set amount of time.

**Comment by 0-vortex** [comment]
We are using [mtfoley/pr-compliance-action](https://github.com/mtfoley/pr-compliance-action) over in [@open-sauced](https://github.com/open-sauced/open-sauced) but we also have conventional tooling. Would be up to help the tooling effort or helping create a downloadable book out of this repo. What do you think?

**Comment by goldbergyoni** [comment]
@josh-hemphill Love the idea.

@0-vortex Welcome aboard. We would definitely appreciate such help. Are you happy with the pr-compliance bot and it satisfies the requirement laid down here by Josh?

About downloadable guide -Note that @rluvaton and @josh-hemphill and @anediaz already started and surely can use your help as well

**Comment by goldbergyoni** [comment]
@0-vortex If this is still relevant, could be awesome to have a bot guarding our PR
